<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNavBarForm</class>
 <widget class="QWidget" name="frmNavBarForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widgetLeft" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="NavBar" name="navBar" native="true">
        <property name="lineWidth" stdset="0">
         <number>5</number>
        </property>
        <property name="horizontal" stdset="0">
         <bool>false</bool>
        </property>
        <property name="flat" stdset="0">
         <bool>true</bool>
        </property>
        <property name="radius" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="wMain">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="MyCustomPlotWidget" name="wUser">
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="MySerialWidget" name="wSerialSet">
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </widget>
     <widget class="ThemeSelection" name="wTheme"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavBar</class>
   <extends>QWidget</extends>
   <header>navbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MySerialWidget</class>
   <extends>QWidget</extends>
   <header>myserialwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyCustomPlotWidget</class>
   <extends>QWidget</extends>
   <header>mycustomplotwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ThemeSelection</class>
   <extends>QWidget</extends>
   <header>themeselection.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
